deal.II-toolchain/packages/dealii.package

```diff
@@ -136,10 +135,9 @@ fi
 
 if [ ! -z "${P4EST_DIR}" ]; then
     cecho ${INFO} "deal.II: configuration with P4EST"
-    CONFOPTS="\
-        ${CONFOPTS} \
-        -D DEAL_II_WITH_P4EST:BOOL=ON \
-        -D P4EST_DIR=${P4EST_DIR}"
+    CONFOPTS="${CONFOPTS} \
+      -D DEAL_II_WITH_P4EST:BOOL=ON"
+      -D P4EST_DIR=${P4EST_DIR}"
```

Comment:
This doesn't look right. Where is the \?

Reply:
Yeah sorry ....

Reply:
and the additional trailing quote in 139 shouldn't be there

Reply:
Ah, was resolved already